recipes/cleo/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cleo" %}
+{% set version = "0.6.8" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "85a63076b72ca376fb06668be1fc7758dc16740b394783d5cc65200c4b32f71b" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - pastel >=0.1.0,<0.2.0
+    - pylev >=1.3,<2.0
+  run:
+    - python
+    - pastel >=0.1.0,<0.2.0
+    - pylev >=1.3,<2.0
+
+test:
+  imports:
+    - cleo
+    - cleo.commands
+    - cleo.commands.completions
+    - cleo.descriptors
+    - cleo.exceptions
+    - cleo.formatters
+    - cleo.helpers
+    - cleo.inputs
+    - cleo.outputs
+    - cleo.questions
+    - cleo.styles
+    - cleo.testers
+    - cleo.validators
+
+about:
+  home: https://github.com/sdispater/cleo
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Cleo allows you to create beautiful and testable command-line interfaces.
+  description: Create beautiful and testable command-line interfaces.
+  doc_url: http://cleo.readthedocs.io/en/latest/
+  dev_url: https://github.com/sdispater/cleo
+
+extra:
+  recipe-maintainers:
+    - nick-youngblut
```

Comment:
happy to help you here, I'm not sure about the others, they usually are for r-* packages